roles/confluent.test/molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml

```
@@ -0,0 +1,261 @@
+
+
+
+      cluster2:
+        ssl_enabled: true
+        ssl_custom_certs: true
+        # Paths relative to all.yml
+        ssl_ca_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        ssl_signed_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        ssl_key_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        ssl_key_password: keypass
+        sasl_protocol: kerberos
+
+        zookeeper_kerberos_principal: "zookeeper/{{inventory_hostname}}.confluent@{{kerberos.realm | upper}}"
+        zookeeper_kerberos_keytab_path: "roles/confluent.test/molecule/{{scenario_name}}/keytabs/zookeeper-{{inventory_hostname}}.keytab"
+
+        kafka_broker_kerberos_principal: "{{kerberos_kafka_broker_primary}}/{{inventory_hostname}}.confluent@{{kerberos.realm | upper}}"
+        kafka_broker_kerberos_keytab_path: "roles/confluent.test/molecule/{{scenario_name}}/keytabs/kafka_broker-{{inventory_hostname}}.keytab"
+
+      kafka_connect_replicator:
+        kafka_connect_replicator_white_list: test-replicator-source
+        kafka_connect_replicator_bootstrap_servers: kafka-broker1:9092
+        kafka_connect_replicator_kerberos_principal: "replicator/kafka-connect-replicator1.confluent@{{kerberos.realm | upper}}"
+        kafka_connect_replicator_kerberos_keytab_path: "roles/confluent.test/molecule/{{scenario_name}}/keytabs/kafka-connect-replicator1.keytab"
+        kafka_connect_replicator_ssl_enabled: true
+        kafka_connect_replicator_ssl_ca_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        kafka_connect_replicator_ssl_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        kafka_connect_replicator_ssl_key_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        kafka_connect_replicator_ssl_key_password: keypass
+
+        kafka_connect_replicator_consumer_bootstrap_servers: mds-kafka-broker1:9092
+        kafka_connect_replicator_consumer_sasl_plain_username: kafka_connect_replicator
+        kafka_connect_replicator_consumer_sasl_plain_password: kafka_connect_replicator-secret
+        kafka_connect_replicator_consumer_ssl_enabled: true
+        kafka_connect_replicator_consumer_ssl_ca_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        kafka_connect_replicator_consumer_ssl_cert_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        kafka_connect_replicator_consumer_ssl_key_path: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        kafka_connect_replicator_consumer_ssl_key_password: keypass
```

so where these vars are defined you should default them to the first set:
```suggestion
        kafka_connect_replicator_consumer_ssl_ca_cert_path: "{{kafka_connect_replicator_ssl_ca_cert_path}}"
        kafka_connect_replicator_consumer_ssl_cert_path: "{{kafka_connect_replicator_ssl_cert_path}}"
        kafka_connect_replicator_consumer_ssl_key_path: "{{kafka_connect_replicator_ssl_key_path}}"
        kafka_connect_replicator_consumer_ssl_key_password: "{{kafka_connect_replicator_ssl_key_password}}"
```

and then the user would only have to define the first set of vars

THIS does not void the good work that you have where they can have multiple sets of keystores/truststores. it only means they would have to set extra vars IF you want two sets